update-all-examples.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
-SDJWT_ARGS="--replace-examples-in main.md --nonce XZOUco1u_gEPknxS78sWWg --iat 1516239022 --exp 1516247022 --no-randomness"
+SDJWT_ARGS="-d DEBUG --replace-examples-in main.md --nonce XZOUco1u_gEPknxS78sWWg --iat 1516239022 --exp 1516247022 --no-randomness"
 
-sd_jwt sd_jwt/examples/simple.yml $SDJWT_ARGS && \
-sd_jwt sd_jwt/examples/simple_structured.yml $SDJWT_ARGS && \
-sd_jwt sd_jwt/examples/complex.yml $SDJWT_ARGS
+for file in sd_jwt/examples/*.yml
```

Comment:
it could be a pain if we decide to push many examples but use only some of them for updating the main.md

a huge stdout is a problem

probably I'll push the eidas example I'm working on but I would tell to users that they may use them without enabling all of them by default

Reply:
In 'replace' mode, the stdout is mainly for placeholders that have not been found, which is really not that much.

Example right now:
```
danielf@www:~/git/ietf/draft-selective-disclosure-jwt$ ./update-all-examples.sh 
Using fixed randomness for demo purposes
Could not find placeholder with id example-simple-serialized_svc
Could not find placeholder with id example-simple-verified_contents
Using fixed randomness for demo purposes
Could not find placeholder with id example-simple_structured-user_claims
Could not find placeholder with id example-simple_structured-serialized_sd_jwt
Could not find placeholder with id example-simple_structured-serialized_svc
Could not find placeholder with id example-simple_structured-combined_sd_jwt_svc
Could not find placeholder with id example-simple_structured-serialized_sd_jwt_release
Could not find placeholder with id example-simple_structured-combined_sd_jwt_sd_jwt_release
Could not find placeholder with id example-simple_structured-verified_contents
Using fixed randomness for demo purposes
Could not find placeholder with id example-complex-svc_payload
Could not find placeholder with id example-complex-serialized_svc
Could not find placeholder with id example-complex-combined_sd_jwt_svc
Could not find placeholder with id example-complex-serialized_sd_jwt_release
Could not find placeholder with id example-complex-combined_sd_jwt_sd_jwt_release
Could not find placeholder with id example-complex-verified_contents
```